main.ts

```diff
@@ -98,7 +98,8 @@ async function findLinksFromFiles(directory: string) {
     fileCount++;
 
     const filtered = filterLinksFromTokens(tokens);
-    for (const link of filtered) {
+    for (let link of filtered) {
+      link = decodeURI(link);
```

Comment:
nit: feels like it should be done in https://github.com/grammyjs/link-checker/blob/502d305fcbae274a40ee9a2054b5c076e6b5a2bb/utilities.ts#L143